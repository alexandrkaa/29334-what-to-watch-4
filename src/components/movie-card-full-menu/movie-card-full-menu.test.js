import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardFullMenu from './movie-card-full-menu.jsx';

const mockTabs = [
  {
    id: `TAB1`,
    name: `Tab1`,
    isActive: true,
  },
  {
    id: `TAB2`,
    name: `Tab2`,
    isActive: false,
  },
  {
    id: `TAB3`,
    name: `Tab3`,
    isActive: false,
  },
];

it(`<MovieCardFullMenu /> should match snapshot`, () => {
  const onClickMovieCardFullMenuTab = jest.fn();
  const tree = renderer
    .create(
        <MovieCardFullMenu
          tabs={mockTabs}
          onClickMovieCardFullMenuTab={onClickMovieCardFullMenuTab}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
