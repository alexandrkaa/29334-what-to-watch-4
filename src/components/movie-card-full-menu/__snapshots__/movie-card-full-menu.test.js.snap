// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardFullMenu /> should match snapshot 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Tab1
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Tab2
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Tab3
      </a>
    </li>
  </ul>
</nav>
`;
